code/_compile_options.dm

```diff
@@ -65,7 +65,7 @@
 //#define REFERENCE_TRACKING_DEBUG
 
 ///Run a lookup on things hard deleting by default.
-//#define GC_FAILURE_HARD_LOOKUP
+#define GC_FAILURE_HARD_LOOKUP
 #ifdef GC_FAILURE_HARD_LOOKUP
 #define FIND_REF_NO_CHECK_TICK
```

Comment:
You're running this with no check tick? madman

Reply:
Oups, fixed